### PR TITLE
Update subway.json

### PR DESCRIPTION
**Pull request by kjonosm**
update German subway networks:

- delete wrong operator tags
- make network names and short names consistent
- add wikidata
- add match names

### CODE REVIEW

File: data/transit/route/subway.json [hunk 1]
```diff
@@ -355,19 +363,15 @@
...
+      "matchNames": ["vrr"],
       "tags": {
-        "network": "VRR",
+        "network": "Verkehrsverbund Rhein-Ruhr",
+        "network:short": "VRR",
         "network:wikidata": "Q448199",
         "network:wikipedia": "de:Verkehrsverbund Rhein-Ruhr",
-        "operator": "VRR",
-        "operator:wikidata": "Q448199",
-        "operator:wikipedia": "de:Verkehrsverbund Rhein-Ruhr",
```
Comment (peternewman) [inline data/transit/route/subway.json:370]: Is the actual operator known for this subway?

File: data/transit/route/subway.json [hunk 2]
```diff
@@ -256,12 +260,15 @@
       }
     },
     {
-      "displayName": "RMV",
+      "displayName": "Rhein-Main-Verkehrsverbund",
       "id": "rmv-447f34",
       "locationSet": {"include": ["de"]},
+      "matchNames": ["rmv"],
```
[the remark this answers is not in the document]
Comment (bhousel) [reply data/transit/route/subway.json:266]: > @bhousel should we make it automatically put network:short in matchNames too to save all this duplication?

It's redundant in `matchNames` if it's already in `network:short` - I believe the build script will remove the matchName next time I run it.

File: data/transit/route/subway.json (hunk 1 above)
Comment (kjonosm) [reply data/transit/route/subway.json:370]: There are several operators within the network: BOGESTRA, Ruhrbahn, Rheinbahn, DVG. For more details see:
https://ptna.openstreetmap.de/results/DE/NW/DE-NW-VRR-Analysis.html#A2.3.14

File: data/transit/route/subway.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (kjonosm) [reply data/transit/route/subway.json:266]: ok. good to know.

File: data/transit/route/subway.json (hunk 1 above)
Comment (peternewman) [reply data/transit/route/subway.json:370]: Ah okay

**Pull request merged**